src/screens/LandingScreen.js

```diff
@@ -29,22 +31,18 @@ const LandingScreen = ({ navigation }) => {
 		navigation.replace(NAVIGATORS.MAIN);
 	};
 
-	const onAuthStateChanged = async (user) => {
-		if (user) {
-			await loadInitialPlaylists();
-		} else {
-			Alert.alert("Something went wrong please try again");
-		}
+	const handleAnonymousSignIn = async () => {
+		const anonymousUser = await auth().signInAnonymously();
+		dispatch(setUser(anonymousUser));
+		await loadInitialPlaylists();
 	};
 
 	useEffect(() => {
-		const signInUserAnonymously = async () => {
-			await auth().signInAnonymously();
-		};
-
-		const subscriber = auth().onAuthStateChanged(onAuthStateChanged);
-		signInUserAnonymously();
-		return subscriber;
+		if (!user) {
```

Comment:
If the user isn't anonymous you will anonymously sign them in, and then load the initial playlists.